src/data/finset/pointwise.lean

```diff
@@ -160,7 +163,11 @@ lemma mem_mul {x : α} : x ∈ s * t ↔ ∃ y z, y ∈ s ∧ z ∈ t ∧ y * z
 lemma coe_mul (s t : finset α) : (↑(s * t) : set α) = ↑s * ↑t := coe_image₂ _ _ _
 
 @[to_additive] lemma mul_mem_mul : a ∈ s → b ∈ t → a * b ∈ s * t := mem_image₂_of_mem
-@[to_additive] lemma mul_card_le : (s * t).card ≤ s.card * t.card := card_image₂_le _ _ _
+@[to_additive] lemma card_mul_le : (s * t).card ≤ s.card * t.card := card_image₂_le _ _ _
+
+@[to_additive] lemma card_mul_iff :
```

Comment:
```suggestion
@[to_additive] lemma card_mul_eq_iff_inj_on :
```
to match `card_image_eq_iff_inj_on`

Reply:
Renaming `card_image_eq_iff_inj_on` to `card_image_iff`.